git-secrets

```diff
@@ -54,10 +54,10 @@ load_patterns() {
 }
 
 load_allowed() {
-  git config --get-all secrets.allowed
+  git config --get-all secrets.allowed | awk 'NF && $1!~/^#/'
```

Comment:
I don't think we need this filtering on git config values -- they can't contain comments, and I don't think they can be empty. Is there a need to filter here?